### PR TITLE
testimonials

### PR DESCRIPTION
**Pull request by thejamielin**

### CODE REVIEW

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
- For screen widths >= 1800 (xxxl) add more left / right margin around the image so everything is more centered
Screen Shot 2021-11-28 at 8 49 19 AM
- text is too long for for box on iphone 5 (I can fix the red svg that's overlapping since that was from faq) 
Screen Shot 2021-11-28 at 8 51 00 AM
 - Add more space at the top on screensizes >= 1200 (xl)
Screen Shot 2021-11-28 at 8 52 44 AM
 - For screensizes <= 380px (xxs) make the bottom spacing a bit more (especially for iphone 5 its a bit close) 
Screen Shot 2021-11-28 at 8 54 38 AM
- on mobile, move the blue planet svg to the same spot as on the design (notice how its kind of cut off a bit higher up) 
Screen Shot 2021-11-28 at 8 56 11 AM
Screen Shot 2021-11-28 at 8 56 39 AM

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
- Remove margin for image at smaller screen sizes 
- href?

File: src/components/sections/testimonials.jsx [hunk 1]
```diff
@@ -35,23 +62,32 @@ const Testimonials = props => {
...
-        </Carousel>
-      </div> */}
-    </div>
+    <section id="testimonials" className="testimonials__container">
+      <h2 className="testimonials__title">Testimonials</h2>
+      <div className="testimonials__box">
+        <TestimonialsItem item={selected} />
+      </div>
+      <div className="planet_selection_container">
+        {console.log(selected.index)}
```
Comment (ghost) [inline src/components/sections/testimonials.jsx]: delete

File: src/components/sections/testimonials.jsx [hunk 2]
```diff
@@ -1,29 +1,56 @@
...
 
 const testimonialInfo = [
   {
-    img: Exploration,
+    img: BenHenderson,
     author: 'Ben Henderson',
     description:
-      ' It was my first Hackathon, and I had a really great time - if you\'ve never done one before, this is a great choice. Really good environment for people who want to learn something new, meet new people, and/or challenge themselves. It\'s basically what you make of it, and there was plenty of support available if you have no idea what you\'re doing.'
+      " It was my first Hackathon, and I had a really great time - if you've never done one before, this is a great choice. Really good environment for people who want to learn something new, meet new people, and/or challenge themselves. It's basically what you make of it, and there was plenty of support available if you have no idea what you're doing.",
+    index: TESTIMONIAL_PLANETS.MERCURY
```
Comment (ghost) [inline src/components/sections/testimonials.jsx:31]: rename this field to `type`

File: src/components/sections/testimonials.jsx [hunk 3]
```diff
@@ -35,23 +62,32 @@ const Testimonials = props => {
...
+        {console.log(selected.index)}
+        {selected.index === TESTIMONIAL_PLANETS.MERCURY ? <TestimonialsMercuryBright className="testimonials__planet" /> : <TestimonialsMercuryDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.MERCURY))}/> }
+        {selected.index === TESTIMONIAL_PLANETS.VENUS ? <TestimonialsVenusBright className="testimonials__planet" /> : <TestimonialsVenusDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.VENUS))}/> }
+        {selected.index === TESTIMONIAL_PLANETS.NEPTUNE ? <TestimonialsNeptuneBright className="testimonials__planet" /> : <TestimonialsNeptuneDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.NEPTUNE))}/>}
+        {selected.index === TESTIMONIAL_PLANETS.MARS ? <TestimonialsMarsBright className="testimonials__planet" /> : <TestimonialsMarsDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.MARS))}/> }
```
Comment (ghost) [inline src/components/sections/testimonials.jsx:78]: abstract this. can do something like `TESTIMONIALS.map(testimonial => <TestimonialsPlanet testimonial={testimonial} selected={selected.type === testimonial.type} setSelected={setSelected} />`

then in testimonialsPlanet, do a switch case or series of if-statements on which component to return.

File: src/components/testimonials-items/testimonial-item.jsx [hunk 4]
```diff
@@ -1,13 +1,13 @@
-import React, { useState } from 'react';
+import React from 'react';
 
-const TestimonialsItem = ({ img, author, description }) => {
+const TestimonialsItem = ({ item }) => {
```
Comment (ghost) [inline src/components/testimonials-items/testimonial-item.jsx:3]: if the props are the same, and are just wrapped by this object, you may as well just prop spread? at least, not sure why this change is needed

prop spread means where this is called, do `<TestimonialsItem {...item} />` and revert these changes.

File: src/styles/home.scss [hunk 5]
```diff
@@ -220,39 +220,68 @@
...
   }
   @media (min-width: $md) {
-    height: 1550px;
+    padding-top: 200px;
+    height: 2000px;
   }
   @media (min-width: $lg) {
-    height: 1750px;
+    height: 2150px;
   }
```
Comment (ghost) [inline src/styles/home.scss:279]: any reason why the FAQ styling was changed?

File: src/styles/svgs.scss [hunk 6]
```diff
@@ -196,10 +203,39 @@
   @media (min-width: $xl) {
     width: 10em;
     right: 450px;
-    top: 1100px;
+    top: 1200px;
   }
   @media (min-width: $xxl) {
-    top: 1100px;
+    top: 1200px;
```
Comment (ghost) [inline src/styles/svgs.scss:209]: same w this

**Review by ghost — APPROVED** [review]
few small nits

File: src/styles/home.scss (hunk 5 above)
Comment (thejamielin) [reply src/styles/home.scss:279]: I was fixing the padding between the Testimonials section and the FAQ section. I changed it since I couldn't get bottom-padding to change the spacing between the two sections by only changing the Testimonial section. I can fix that.

Comment (jzhzhang66) [reply src/styles/home.scss:279]: WAIT @bugsalexander @thejamielin yes I pushed some additional changes for that after Jamie finished because I realized the container needs to be taller otherwise if you expand the longest question the bottom will get cut out. So let's keep this as is.

File: src/components/sections/testimonials.jsx (hunk 1 above)
Comment (thejamielin) [reply src/components/sections/testimonials.jsx:75]: deleted!

File: src/styles/home.scss (hunk 5 above)
Comment (ghost) [reply src/styles/home.scss:279]: sounds good!

**Comment by ghost** [comment]
i gave you an approval, so feel free to merge. fixing nits is optional
